bitnami/mongodb/templates/replicaset/svc.yaml

```diff
@@ -25,6 +25,7 @@ metadata:
   {{- end }}
 spec:
   type: ClusterIP
+  publishNotReadyAddresses: true
```

Comment:
Instead of always setting the parameter to true, can you parameterize it (with default to `false`)? This way we maintain the current behavior and it can be set in an easy way when required.

Reply:
I can do that. But I wonder, what's the use case for the current behavior - could changing the default false -> true potentially be a breaking change?

Reply:
The thing is that we have not being reported with any issue related to current setup, so to be on the safe side I think it is worth it to keep it this way for now yet allowing users to customize it.